impl/al.hpp

```diff
@@ -2,5 +2,6 @@
 #define OPENALPP_AL_HPP
 
 #include "AL/al.h"
+#include <AL/alext.h>
```

Comment:
Please use consistent quotes/brackets for includes
```suggestion
#include "AL/alext.h"
```